libcodechecker/analyze/store_handler.py

```diff
@@ -0,0 +1,526 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from hashlib import sha256
+import json
+import os
+import tempfile
+import zlib
+
+import sqlalchemy
+
+import shared
+from codeCheckerDBAccess.ttypes import *
+
+from libcodechecker.analyze import skiplist_handler
+from libcodechecker.logger import LoggerFactory
+from libcodechecker.orm_model import *
+
+LOG = LoggerFactory.get_new_logger('STORE HANDLER')
+
+
+def process_compile_db(compile_db):
```

Comment:
Maybe a more descriptive name like `make_absolute_path_compile_db`. Process is overly generic in my opinion.